### ISSUE TITLE
Changing to less crashes the default build

### ISSUE THREAD

**Issue by ImBeCiliC** [preactjs/preact-cli#1173]
**Do you want to request a _feature_ or report a _bug_?**
_bug_
**What is the current behaviour?**
Creating a new default Project with preact-cli install less and less-loader. Change all .css to .less.
Start the dev server which cashes.

**If the current behaviour is a bug, please provide the steps to reproduce.**

1. `preact create default less-test`
2. `npm install --save-dev less less-loader`
3. `npm run dev`
4. run dev should fail to build and start.

**What is the expected behaviour?**
installing less on a clean project should work with the steps described in the docs

**If this is a feature request, what is motivation or use case for changing the behaviour?**
----
**Please mention other relevant information.**

Please paste the results of `preact info` here.

**Comment by psabharwal123** [comment]
Please paste the results of preact info here.
^ would be helpful

**Comment by ForsakenHarmony** [comment]
Also logs of what the error is

**Comment by ImBeCiliC** [comment]
To break it down. less-loader introduced another variable in v6 which is not handled by the Proxy Loader. Check the less-loader breaking changes for that. 

I solved the issue by downgrading the less-loader to v5. I guess you solved it in the v3 by upgrading the packages but this should be verified. 

I was running on preact-cli 2.2.1 cause I couldn't install v3 for some reason. Tried multiple ways on Mac.
